tests/test_lab_dev/test_circuit_components_utils/test_b_to_w.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for BtoW."""
+
+# pylint: disable=fixme, missing-function-docstring, pointless-statement
+
+import numpy as np
+import pytest
+
+from mrmustard import math
+from mrmustard.lab_dev import BtoW, Dgate, Ket
+from mrmustard.physics.wigner import wigner_discretized
+
+
+class TestBtoW:
```

Comment:
best to add a test where we mess with hbar. this version if I understand correctly should be independent of hbar (taken as a function of z and z*), but when one refers to q and p then it should depend on it.